modules/capability/module.go

```diff
@@ -26,7 +26,6 @@ import (
 
 var (
 	_ module.AppModule           = (*AppModule)(nil)
-	_ module.AppModuleBasic      = (*AppModuleBasic)(nil)
```

Comment:
note that theres associated functions that can additionally be removed, currently they should be duplicated (e.g `Name()` IIRC)

Reply:
Will incorporate this in another PR, for now edited the issues addressed and reverted changes